gRPC for WCF Index content (#15586)
dotnet/docs: docs/architecture/grpc-for-wcf-developers/index.md

---
title: ASP.NET Core gRPC for WCF Developers - gRPC for WCF Developers

> description: Introduction to building gRPC Services in ASP.NET Core 3.0 with particular reference to WCF

Reviewer: ```suggestion
description: Introduction to building gRPC Services in ASP.NET Core 3.0 for WCF developers
```
